pelican/__init__.py

```diff
@@ -467,7 +467,13 @@ def main():
                             ', '.join(k for k, v in modified.items() if v)))
 
                         if modified['content'] is None:
-                            logger.warning('No valid files found in content.')
+                            reader_descs = set(['%s (%s)' % (type(r).__name__, 
+                                                             ', '.join(r.file_extensions)) 
+                                                for r in readers.readers.values()
+                                                if r.enabled])
+                            
+                            logger.warning('No valid files found in content for the active readers:\n' +
+                                           '\n'.join(reader_descs))
```

Comment:
```python
'\n'.join(sorted(reader_descs))
```
would reduce the potential randomness in order coming from `set`

Otherwise it's good to go.